### ISSUE TITLE
Feature request: add a callable refresh function

### ISSUE THREAD

**Issue by sforli** [Textualize/frogmouth#44]
It would be great to have a way to refresh the content of a file using a key combo but also using some kind of programatic way (socket? external command?). This would be perfect for using the viewer in a preview splits with text editors like Vim and/or Tmux environments.

**Comment by davep** [comment]
A refresh key at the very least does seem like a good idea.

**Comment by sforli** [comment]
Another interesting approach would be to monitor the file for changes, either using a cheap timed loop or something more sophisticated like [watchdog](https://pythonhosted.org/watchdog/).

**Comment by davep** [comment]
Yup, something like that would be the plan. Ideally first I want us to make some enhancements to the Textual Markdown widget that would help with this working really well (things that would help with keeping position in the document so it doesn't keep jumping back to the top, etc).
